composer/packages/theme/src/definitions/definitions.config

```diff
@@ -0,0 +1,15 @@
+/*******************************
+            Folders
+*******************************/
+
+/* Path to theme packages, relative to node_modules/sementic-ui */
+@themesFolder : '../../packages/theme/src/themes';
+
+/* Path to site override folder, relative to node_modules/sementic-ui */
+@siteFolder  : '../../packages/theme/src/themes';
+
+/*******************************
+         Import Theme
+*******************************/
+
+@import "~semantic-ui-less/theme.less";
```

Comment:
Can we end with new line for files? WDYT?

Reply:
There is a new line